cdk/lib/rds-stack.ts

```diff
@@ -1,5 +1,7 @@
 require("dotenv").config();
```

Comment:
If we're removing the ARN from the file and moving it to the root in bin/api.ts can you remove instances of require("dotenv").config(); and add that to bin/api.ts. It's no longer needed if we're not pulling env variables.